backend/api/Areas/Contacts/Models/Contact/OrganizationModel.cs

```diff
@@ -7,39 +9,49 @@ public class OrganizationModel
     {
         #region Properties
         /// <summary>
-        /// get/set - The primary key to identify the organization.
+        /// get/set - The organization's id.
         /// </summary>
         public long Id { get; set; }
 
         /// <summary>
-        /// get/set - The organization's company name.
+        /// get/set - The organization's disabled status flag.
+        /// </summary>
+        public bool IsDisabled { get; set; }
+
+        /// <summary>
+        /// get/set - The organization's concatenated name.
```

Comment:
concatenated name? what do you mean? isn't this a discrete field?

Reply:
good catch, it came from the person's name.